doc/lightning_kokkos/device.rst

```diff
@@ -0,0 +1,77 @@
+Lightning Kokkos device
+=======================
+
+The `lightning.kokkos` device can run using a variety of HPC-focused backends, including GPUs,
+enabling accelerated simulation of quantum state-vector evolution.
+
+A ``lightning.kokkos`` device can be loaded using:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.kokkos", wires=2)
+
+The ``lightning.kokkos`` device also directly supports quantum circuit gradients using the adjoint differentiation method. This can be enabled at the PennyLane QNode level with:
+
+.. code-block:: python
+
+    qml.qnode(dev, diff_method="adjoint")
+    def circuit(params):
+        ...
+
+Check out the :doc:`/lightning_kokkos/installation` guide for more information.
+
+Supported operations and observables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Supported operations:**
+
+.. raw:: html
+
+    <div class="summary-table">
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.BasisState
```

Comment:
There are more supported operations, but I guess this is not meant to be an exhaustive list?